pkg/registry/core/pod/strategy.go

```diff
@@ -975,3 +984,18 @@ func apparmorFieldForAnnotation(annotation string) *api.AppArmorProfile {
 	// length or if the annotation has an unrecognized value
 	return nil
 }
+
+// setGenerationForPodSpecUpdate bumps metadata.generation if needed for any updates
+// to the podspec.
+func setGenerationForPodSpecUpdate(newPod, oldPod *api.Pod) {
+	// Mirror pods should not have Generation set.
+	if metav1.HasAnnotation(newPod.ObjectMeta, v1.MirrorPodAnnotationKey) {
+		newPod.Generation = 0
```

Comment:
same comment about not special-casing mirror pods

Reply:
fixed